.rubocop.yml

```diff
@@ -3,47 +3,6 @@ AllCops:
     - 'data/**/*'
     - 'vendor/**/*'
```

Comment:
I wonder if this line should be in `.rubocop_base.yml` as well? That would avoid problems with Travis installing gems into `vendor/`.